assets/dev/js/editor/elements/types/container.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Is this file in use?
+// eslint-disable-next-line import/no-unresolved
```

Comment:
@iNewLegend can you check?

Reply:
Not in use, but in `3.7.0` it's in use, how it reached develop?